### ISSUE TITLE
newser.com

### ISSUE THREAD

**Issue by 8k1uq2u27gliyobfc** [reek/anti-adblock-killer#3492]
http://imgur.com/a/iUQ5A
Mozilla Firefox
Tampermonkey
Adblock Plus
EasyList + AakList

**Comment by uBlock-user** [comment]
Can't reproduce.

**Comment by uBlock-user** [comment]
But I rather see nasty ads being inserted

![](https://i.gyazo.com/a25a019a6dee9e6e20c57cb102afd6ec.png)

**Comment by uBlock-user** [comment]
It's Yavli shit. but `yavli-defuser` isn't working anymore.

**Comment by uBlock-user** [comment]
It's not on the homepage but rather on articles - http://www.newser.com/story/247183/trump-lashes-out-at-ceo-who-criticized-va-response.html

**Comment by jspenguin2017** [comment]
Does this work reliably? 
```
newser.com###divRightRail > div:nth-child(2)
newser.com###divStoryColumn > div:nth-child(3)
```

**Comment by uBlock-user** [comment]
Yeah that seems to work

**Comment by uBlock-user** [comment]
Hey @okiehsch does `yavli-defuser` work on this ?

**Comment by okiehsch** [comment]
No, it does not.
`newser.com##script:inject(abort-on-property-read.js, oio)`

**Comment by uBlock-user** [comment]
Yeah they work, but now they're conflicting with uBlock Protector

**Comment by jspenguin2017** [comment]
@uBlock-user Can't reproduce.

**Comment by jspenguin2017** [comment]
I'll remove the rule through, since it is in uBlock filters now.

**Comment by jspenguin2017** [comment]
Eh... the pipeline choked up... I guess the callback server is absolutely required, can't cheap out on that...

**Comment by uBlock-user** [comment]
I can reproduce and it's because of that checkads you blew open and along with okiehsch's filter, they clash and the ads appear again. I'm Chromium 62.

**Comment by uBlock-user** [comment]
You have these filters too that's why you can't reproduce.

```
newser.com###divRightRail > div:nth-child(2)
newser.com###divStoryColumn > div:nth-child(3)
```

Remove those two and just keep 'checkads' and uBlock Filters's filter.

**Comment by jspenguin2017** [comment]
Well, yea, it's the way uBO combines script rules, and uBlock Protector is always faster which crashes the script from uBO. 
I'll update the store version eventually, it's going to take a while though, my server broke (again), either I'm a terrible sysadmin or my hardware stinks. I guess I'll try hardware managed by other people. Considering DigitalOcean and Linode for now. I guess I'll try Linode first, it seems cheaper. 
There is a slight issue is that the box has only 1GB of RAM, that's for operating system plus server softwares. It wasn't a problem before because my network would have choked up well before PHP filled Pi's RAM, but now it's different.
